roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -114,3 +114,37 @@ vault_deployment_type: docker
 k8s_image_pull_policy: IfNotPresent
 efk_enabled: false
 enable_network_policy: false
+
+# Configure proxy environment variable map
+default_no_proxy_list: "\
+  127.0.0.1,127.0.1.1,127.1.1.1,localhost,\
+  {{kube_service_addresses}},{{kube_pods_subnet}},\
+  {{apiserver_loadbalancer_domain_name|default('lb-apiserver.kubernetes.local')}},\
+  .{{cluster_name}}"
+
+bypass_proxy_for_cluster_hosts: true
+proxy_env: "{
```

Comment:
isn't this a string and not a dict ?

Reply:
strings in json format is converted by ansible to the corresponding data types (dict, list)